rust-toolchain

```diff
@@ -1,4 +1,3 @@
 [toolchain]
-channel = "nightly-2023-03-15"
-components = []
+channel = "1.72"
```

Comment:
```suggestion
channel = "1.72.0"
```

Reply:
If you define just `1.72`, then will be installed latest patch version of `1.72.x`. E.g. now it will be `1.72.1`. It'd better to add a patch version.